### PR TITLE
commit 1 b1 github2

### PR DESCRIPTION
**Pull request by razvaniuoras**
Added commit 1 b1 github2 +modified line1

### CODE REVIEW

File: file1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-commit 1 master github1 
\ No newline at end of file
+commit 1 master github1-github2
```
Comment (iuorasrazvan) [inline file1.txt]: better github11

Comment (iuorasrazvan) [inline file1.txt]: If you make the change i ll be happy to merge it.

**Comment by iuorasrazvan** [comment]
ok

**Comment by iuorasrazvan** [comment]
If you make the change i ll be happy to merge it.

File: file1.txt (hunk 1 above)
Comment (iuorasrazvan) [inline file1.txt]: make here github222 instead of github2

**Comment by iuorasrazvan** [comment]
If you make the change i will merge.

**Comment by razvaniuoras** [comment]
Anything else ?

**Comment by iuorasrazvan** [comment]
I merged mister.

**Pull request merged**